nx/Makefile

```diff
@@ -24,14 +24,14 @@ VERSION	:=	$(LIBNX_MAJOR).$(LIBNX_MINOR).$(LIBNX_PATCH)
 #---------------------------------------------------------------------------------
 TARGET		:=	nx
 #BUILD		:=	build
-SOURCES		:=	source/arm source/kernel source/services source/nvidia source/nvidia/ioctl source/display source/audio source/applets source/runtime source/runtime/devices source/runtime/util/utf
+SOURCES		:=	source/arm source/kernel source/services source/crypto source/nvidia source/nvidia/ioctl source/display source/audio source/applets source/runtime source/runtime/devices source/runtime/util/utf
 DATA		:=	data
 INCLUDES	:=	include external/bsd/include
 
 #---------------------------------------------------------------------------------
 # options for code generation
 #---------------------------------------------------------------------------------
-ARCH	:=	-march=armv8-a -mtune=cortex-a57 -mtp=soft -fPIC -ftls-model=local-exec
+ARCH	:=	-march=armv8-a+crypto -mtune=cortex-a57 -mtp=soft -fPIC -ftls-model=local-exec
```

Comment:
Is `-march=armv8-a+crypto` the right arch to enable for Cortex-A57? Are there any more arch extensions that should in theory be enabled (such as crc maybe)?

After this PR gets merged I want to update all template makefiles with the new arch selection flag, so it's important we get it right.

Reply:
AFAIK crc is indeed needed

Reply:
https://github.com/switchbrew/nx-hbexploit300-obf/blob/master/installer_obf/Makefile.normal#L42

Reply:
So `-march=armv8-a+crc+crypto` it is then.